docs/v4.0-draft1/version-control.md

```diff
@@ -11,7 +11,7 @@ v4.0-Draft 1 - 25th April 2024
 | PISP | ISO20022 changes<br><ul>`CategoryPurposeCode` added to Risk object</li><li>`ExtendedPurpose` moved to Risk object (used for International payments only)</li><li>`Proxy` added to CreditorAccount/DebtorAccount</li><li>`LEI` added to Creditor/Debtor objects</li><li>`UltimateCreditor` and `UltimateDebtor` added to payments</li><li>`RemittanceInformation` added to payments</li><li>`MandateRelatedInformation` added to domestic and international StandingOrder</li><li>`RegulatoryReporting` added to payments</li></ul> |
 | Errors | <ul><li>Simplified error object, top level `Code` and `Message` have been removed.</li><li>Error codes now in ISO 20022 code value format (4 chars)</li><li>Additional guidance on errors experienced in the redirect flow or from PSU initiated changes</li></ul> |
 | VRP | <ul><li>`OBRisk2` object added without ExtendedPurpose field</li><li>`ContractPresentIndicator` typo fixed</li><li>Added guidance on dynamic reference information for VRP payments</li></ul> |
-| Payments | <ul><li>Updated payment message types information to include ISO 20022 changes, CHAPS migration and CBPR+</li><li>Richer payment status information aligned to ISO 20022 ExternalTransactionStatus1code:<ul><li>`Status` changed to `StatusCode`</li></ul></li><li>New `StatusReason` array containing 0..*:<ul><li>`StatusReasonCode`</li><li>`Path`</li><li>`StatusReasonDescription`</li></ul></li><li>`PaymentDetails` changed to `StatusDetail`, an array of payment status details</li></ul></li></ul>|
+| Payments | <ul><li>Updated payment message types information to include ISO 20022 changes, CHAPS migration and CBPR+</li><li>Richer payment status information aligned to ISO 20022 ExternalTransactionStatus1code:<ul><li>`Status` changed to `Status`</li></ul></li><li>New `StatusReason` array containing 0..*:<ul><li>`StatusReasonCode`</li><li>`Path`</li><li>`StatusReasonDescription`</li></ul></li><li>`PaymentDetails` changed to `StatusDetail`, an array of payment status details</li></ul></li></ul>|
```

Comment:
This isn't correct StatusCode needs reinserting here, needs reverting and a new entry for the rollback adding